lib/googleauth/credentials_loader.rb

```diff
@@ -39,14 +39,17 @@ module Auth
     # credentials files on the file system.
     module CredentialsLoader
       extend Memoist
-      ENV_VAR = 'GOOGLE_APPLICATION_CREDENTIALS'.freeze
-
-      PRIVATE_KEY_VAR = 'GOOGLE_PRIVATE_KEY'.freeze
-      CLIENT_EMAIL_VAR = 'GOOGLE_CLIENT_EMAIL'.freeze
-      CLIENT_ID_VAR = 'GOOGLE_CLIENT_ID'.freeze
-      CLIENT_SECRET_VAR = 'GOOGLE_CLIENT_SECRET'.freeze
-      REFRESH_TOKEN_VAR = 'GOOGLE_REFRESH_TOKEN'.freeze
-      ACCOUNT_TYPE_VAR = 'GOOGLE_ACCOUNT_TYPE'.freeze
+      ENV_VAR                   = 'GOOGLE_APPLICATION_CREDENTIALS'.freeze
+      PRIVATE_KEY_VAR           = 'GOOGLE_PRIVATE_KEY'.freeze
+      CLIENT_EMAIL_VAR          = 'GOOGLE_CLIENT_EMAIL'.freeze
+      CLIENT_ID_VAR             = 'GOOGLE_CLIENT_ID'.freeze
+      CLIENT_SECRET_VAR         = 'GOOGLE_CLIENT_SECRET'.freeze
+      REFRESH_TOKEN_VAR         = 'GOOGLE_REFRESH_TOKEN'.freeze
+      ACCOUNT_TYPE_VAR          = 'GOOGLE_ACCOUNT_TYPE'.freeze
+      PROJECT_ID_VAR            = 'GOOGLE_CLOUD_PROJECT'.freeze
```

Comment:
Does it make sense for googleauth to use environment variables that are specific to the Google Cloud gems? From the other environment variables I would assume this would be `GOOGLE_PROJECT_ID`.